core/src/epicli/data/common/defaults/configuration/logstash.yml

```diff
@@ -0,0 +1,7 @@
+kind: configuration/logstash
+title: "Logstash"
+name: default
+specification:
+  versions:
+    RedHat: "7.8.1"
```

Comment:
This is kinda interesting what happens when an user uses the offline package and changes those versions here. 🤔 I don't think we should allow modifying versions in the config if we don't support it in other places in the same codebase. :-1: For me personally it makes no sense at all. :) I believe this should be set in "defaults" for the role and should not be exposed to the user.

Reply:
That's fine, I will change it!